src/components/LoginRegister.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { useState } from 'react'
+import login_costume from '../assets/images/login-costume.svg'
+import { Link } from 'react-router-dom'
+import { Button, Form } from 'react-bootstrap'
+
+const URL = 'http://localhost:3000'
+export const WebInform = () => {
+  return (
+    <>
+      <h1 className="text-start d-inline bookbix-logo">BookBix</h1>
+      <div className="login-left d-flex">
+        <img
+          src={login_costume}
+          className="img-fluid"
+          alt="costume-of-login-page"
+        />
+      </div>
+    </>
+  )
+}
+
+export const Registration = () => {
+  const [username, setUsername] = useState('')
+  const [email, setEmail] = useState('')
+  const [password, setPassword] = useState('')
+  const [confirmPassword, setConfirmPassword] = useState('')
+  const [type, setType] = useState('')
+  const [error, setError] = useState<string | null>(null)
+
+  const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault()
+    if (!username || !email || !password || !confirmPassword || !type) {
+      setError('Please fill in all fields')
+      return
+    }
+    if (password !== confirmPassword) {
+      setError('Password and Confirm Password must be the same')
+      return
+    }
+    setError(null)
+    try {
+      const response = await fetch(`${URL}/register`, {
```

Comment:
fix the same as above and see the API spec.
```suggestion
      const response = await fetch(`${URL}/register`, {
```